### PR TITLE
(alumni request) Update officer photos, names, and bios.

### PR DESCRIPTION
**Pull request by clariseliu**
i do not know how to render the website, so things might definitely be wrong.

### CODE REVIEW

File: _data/people_currentcrew_officers.json [hunk 1]
```diff
@@ -1,51 +1,76 @@
...
-        "name": "Iris Chang",
-        "andrewid": "ischang",
-        "img": "ischang.jpg",
+        "name": "Grace Kam",
+        "andrewid": "gskam",
+        "img": "gskam.jpg",
+        "title": "Public Relations Chair",
+        "desc": "Grace is a sophomore studying Materials Science and Biomedical Engineering. She hasn't yet chosen a focus within AB Tech, but is interested in all areas. Her hobbies include baking, playing violin, and sleeping."
+    }, 
+
```
Comment (DaAwesomeP) [inline _data/people_currentcrew_officers.json:60]: Please remove this extra line.

File: _data/people_currentcrew_officers.json [hunk 2]
```diff
@@ -1,51 +1,76 @@
...
+    {
+        "name": "Clarise Liu",
+        "andrewid": "crliu",
+        "img": "crliu.jpg",
         "title": "Training Chair",
-        "desc": "Iris is a junior studying Professional Writing and Business with a focus on communication design. This is her third year in AB Tech, where she does stage management. In her free time, she enjoys teaching her parakeets new tricks."
+        "desc": "Clarise is a sophomore studying computer science. They mainly do lighting & event communication in AB Tech but dabbles in other fields as well. Clarise loves music, animated movies, running, and anything colorful that makes sound."
     },
+
```
Comment (DaAwesomeP) [inline _data/people_currentcrew_officers.json:68]: Please remove this extra line.

File: _data/people_currentcrew_officers.json [hunk 3]
```diff
@@ -1,51 +1,76 @@
...
-        "img": "lywu.jpg",
-        "title": "Secretary",
-        "desc": "Laura is a sophomore studying electrical & computer engineering and music tech (but still undecided). In AB Tech, she works with event communication."
-    }
+        "name": "Greta Luo",
+        "andrewid": "gretal",
+        "img": "gretal.jpg",
+        "title": "Recruitment & Retainment Chair",
+        "desc": "Greta is a senior/first year master’s student studying statistics + information systems. In AB Tech, she likes helping out with stage management. She also likes to dance a lot so you might see her choreography (and lighting) in Dancers’ Symposium!"
+    },
```
Comment (DaAwesomeP) [inline _data/people_currentcrew_officers.json:75]: Please remove this trailing comma.

File: _data/people_currentcrew_officers.json [hunk 4]
```diff
@@ -1,51 +1,76 @@
...
-        "title": "Equipment Manager",
-        "desc": "Alex Strasser is junior studying ECE and Robotics. He enjoys repairing our equipment and doing lighting for events."
-    },
+        "name": "Zara Mansoor",
+        "andrewid": "zmansoor",
+        "img": "zmansoor.jpg",
+        "title": "Internal Operations Chair",
+        "desc": "Zara is a sophomore studying Electrical and Computer Engineering. She’s interested in sound in tech but hoping to learn more in other areas as well!"
+    }, 
+
```
Comment (DaAwesomeP) [inline _data/people_currentcrew_officers.json:52]: Please remove this extra line.

**Review by DaAwesomeP — CHANGES_REQUESTED** [review]
Please rename `20220821_173446.jpg` accordingly. See other comments in review.

Otherwise, looks good!

**Comment by clariseliu** [comment]
oof sorry will do this asap

**Comment by clariseliu** [comment]
depression

File: _data/people_currentcrew_officers.json [hunk 5]
```diff
@@ -1,51 +1,72 @@
...
-        "img": "astrasse.jpg",
-        "title": "Equipment Manager",
-        "desc": "Alex Strasser is junior studying ECE and Robotics. He enjoys repairing our equipment and doing lighting for events."
-    },
+        "name": "Zara Mansoor",
+        "andrewid": "zmansoor",
+        "img": "zmansoor.jpg",
+        "title": "Internal Operations Chair",
+        "desc": "Zara is a sophomore studying Electrical and Computer Engineering. She’s interested in sound in tech but hoping to learn more in other areas as well!"
+    },
```
Comment (DaAwesomeP) [inline _data/people_currentcrew_officers.json:50]: Please remove the extra space at end of line.

File: _data/people_currentcrew_officers.json [hunk 6]
```diff
@@ -1,51 +1,72 @@
...
+    }, 
+    {
+        "name": "Clarise Liu",
+        "andrewid": "crliu",
+        "img": "crliu.jpg",
         "title": "Training Chair",
-        "desc": "Iris is a junior studying Professional Writing and Business with a focus on communication design. This is her third year in AB Tech, where she does stage management. In her free time, she enjoys teaching her parakeets new tricks."
-    },
+        "desc": "Clarise is a sophomore studying computer science. They mainly do lighting & event communication in AB Tech but dabbles in other fields as well. Clarise loves music, animated movies, running, and anything colorful that makes sound."
+    },
```
Comment (DaAwesomeP) [inline _data/people_currentcrew_officers.json:64]: Please remove the extra spaces at end of line.

File: _data/people_currentcrew_officers.json [hunk 7]
```diff
@@ -1,51 +1,72 @@
...
+        "title": "Co-Head of Tech",
+        "desc": "Shannon is a senior studying computer science with a minor in music technology. She does sound and production management in AB Tech. In her spare time, she enjoys audiobooks, trying to play instruments, and long walks (at sunset)."
+    {
+        "name":"Alex Strasser",
+        "andrewid": "astrasse",
+        "img": "astrasse.jpg",
+        "title": "Co-Head of Tech",
+        "desc": "Alex Strasser is a senior studying ECE and Robotics. He enjoys repairing our equipment, doing lighting for events, and working with event organizers to make events meet their dreams!"
+    }, 
+
```
Comment (DaAwesomeP) [inline _data/people_currentcrew_officers.json:22]: Please remove the blank line.

File: _data/people_currentcrew_officers.json [hunk 8]
```diff
@@ -1,51 +1,72 @@
...
-        "desc": "Perry is a senior studying art and electrical & computer engineering. His art focuses on electronics, software, and interaction. In AB Tech he does a lot of sound and event management."
+        "desc": "Iris is a Master's student studying Professional Writing with a focus on communication design and executive comms. This is her fourth year on tech, and she does stage and event management. In her free time, she enjoys teaching her parakeets new tricks."
     },
     {
         "name": "Shannon Ding",
         "andrewid": "sding2",
         "img": "sding2.jpg",
+        "title": "Co-Head of Tech",
+        "desc": "Shannon is a senior studying computer science with a minor in music technology. She does sound and production management in AB Tech. In her spare time, she enjoys audiobooks, trying to play instruments, and long walks (at sunset)."
+    {
```
Comment (DaAwesomeP) [inline _data/people_currentcrew_officers.json:15]: Missing closing bracket from previous block (will error).

**Review by DaAwesomeP — CHANGES_REQUESTED** [review]
Sorry, one more small thing preventing this from working (along wit the small minor things). You're missing an opening curly bracket in one spot.

**Comment by DaAwesomeP** [comment]
@clariseliu any updates?

**Review by DaAwesomeP — APPROVED** [review]

**Pull request merged**